spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/resolver/AetherModuleResolver.java

```diff
@@ -88,10 +89,10 @@ public AetherModuleResolver(File localRepository, Map<String, String> remoteRepo
 		}
 		this.localRepository = localRepository;
 		this.remoteRepositories = new LinkedList<>();
-		if (!CollectionUtils.isEmpty(remoteRepositories)) {
-			for (Map.Entry<String, String> remoteRepo : remoteRepositories.entrySet()) {
-				RemoteRepository remoteRepository = new RemoteRepository.Builder(remoteRepo.getKey(),
-						DEFAULT_CONTENT_TYPE, remoteRepo.getValue()).build();
+		if (!CollectionUtils.isEmpty(remoteRepositoriesSet)) {
+			for (String remoteRepo : remoteRepositoriesSet) {
+				RemoteRepository remoteRepository = new RemoteRepository.Builder("",
```

Comment:
Wouldn't it be simpler for the end-user if we allowed a comma-separated list of values that gets converted to a collection/array and we built the map ourselves (using default/generated ids)? Passing properties for binding to a map making this slightly more complicated?